index.js

```diff
@@ -315,8 +311,8 @@ module.exports = function (log, indexesPath) {
 
   const undefinedBipf = bipf.allocAndEncode(undefined)
 
-  function checkEqual(opData, buffer) {
-    const fieldStart = opData.seek(buffer)
+  function checkEqual(opData, buffer, pValue) {
+    const fieldStart = opData.seek(buffer, 0, pValue)
```

Comment:
In what implementations of `opData.seek` will we need the 2nd argument `0`? I took a look at the ssb-db2 PR, and it seems we're never using this 2nd arg

Reply:
It is used here: https://github.com/ssb-ngi-pointer/jitdb/blob/6d62f3acc6935c6a900ed07a8bb805ca6504281b/operators.js#L40. At first I just had pValue as the first argument after buffer, but that broke includes + pluck test ;)